ParseTwitterUtils/PF_Twitter.m

```diff
@@ -92,6 +92,30 @@ - (BFTask *)authorizeInBackground {
         return source.task;
     }];
 }
+- (BFTask *)deauthorizeInBackground {
+    if (self.consumerKey.length == 0 || self.consumerSecret.length == 0) {
+        //TODO: (nlutsenko) This doesn't look right, maybe we should add additional error code?
+        return [BFTask taskWithError:[NSError errorWithDomain:PFParseErrorDomain code:1 userInfo:nil]];
+    }
+
+    return [[self _performDeauthAsync] pftw_continueAsyncWithBlock:^id(BFTask *task) {
+        BFTaskCompletionSource *source = [BFTaskCompletionSource taskCompletionSource];
+        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Not sure whether we need to actually dispatch to main thread here, since the model of tasks is to move to another thread on the next task, but not on this one.
This basically means - that you don't need to `dispatch_async` here, and can simply `setLoginResultValues:`